src/disk_interface.cc

```diff
@@ -100,19 +100,20 @@ bool StatAllFilesInDir(const string& dir, map<string, TimeStamp>* stamps,
       static_cast<FINDEX_INFO_LEVELS>(1);
   FINDEX_INFO_LEVELS level =
       can_use_basic_info ? kFindExInfoBasic : FindExInfoStandard;
-  WIN32_FIND_DATAA ffd;
-  HANDLE find_handle = FindFirstFileExA((dir + "\\*").c_str(), level, &ffd,
-                                        FindExSearchNameMatch, NULL, 0);
+
+  WIN32_FIND_DATA ffd;
+  HANDLE find_handle = FindFirstFileEx(Utf8ToWide(dir + "\\*").c_str(), level,
```

Comment:
String Concatination + allocating and copying the result.

Please consider if there is a better way to do this.

Possibilities include allocating a single `std::wstring` prior to calling Utf8ToWide, and reserving more space than actually required to store the converted `dir` variable, and then appending the "\\*" to the end.